README.md

```diff
@@ -56,49 +69,44 @@ The Apache webserver is configured via the `httpd-modsecurity.conf` file overrid
-| ARG_LENGTH | An integer indicating the arg_length (Default: unlimited) |
-| TOTAL_ARG_LENGTH | An integer indicating the total_arg_length (Default: unlimited) |
-| MAX_FILE_SIZE | An integer indicating the max_file_size (Default: unlimited) |
-| COMBINED_FILE_SIZES | An integer indicating the combined_file_sizes (Default: unlimited) |
-| APACHE_TIMEOUT | Apache integer value indicating the number of seconds before receiving and sending time out (Default: 60) |
-| NGINX_KEEPALIVE_TIMEOUT | Nginx integer value indicating the number of seconds a keep-alive client connection will stay open on the server side (Default: 60) |
-| LOGLEVEL | A string value controlling the number of messages logged to the error_log (Default: warn) |
-| ERRORLOG | A string value indicating the location of the error log file (Default: '/proc/self/fd/2') |
-| PORT | An integer value indicating the port where the webserver is listening to (Default: 80) |
-| USER | A string value indicating the name (or #number) of the user to run httpd as (Default: daemon) |
-| GROUP | Apache string value indicating the name (or #number) of the group to run httpd as (Default: daemon) |
-| SERVERADMIN | A string value indicating the address where problems with the server should be e-mailed (Default: root@localhost) |
-| SERVERNAME | A string value indicating the server name (Default: localhost) |
-| MODSEC_RULE_ENGINE | ModSecurity string value enabling ModSecurity itself (Default: on) |
-| MODSEC_REQ_BODY_ACCESS | ModSecurity string value allowing ModSecurity to access request bodies (Default: on) |
-| MODSEC_REQ_BODY_LIMIT | ModSecurity integer value indicating the maximum request body size  accepted for buffering (Default: 13107200) |
-| MODSEC_RESP_BODY_ACCESS | ModSecurity string value allowing ModSecurity to access response bodies (Default: on) |
-| MODSEC_RESP_BODY_LIMIT | ModSecurity integer value indicating the maximum response body size  accepted for buffering (Default: 524288) |
-| MODSEC_PCRE_MATCH_LIMIT | ModSecurity integer value indicating the limit for the number of internal executions in the PCRE function (Default: 1000) |
-| MODSEC_PCRE_MATCH_LIMIT_RECURSION | ModSecurity integer value indicating the limit for the depth of recursion when calling PCRE function (Default: 1000) |
-| MODSEC_DEFAULT_PHASE1_ACTION | ModSecurity string with the contents for the default action in phase 1 (Default: `'phase:1,log,auditlog,pass,tag:\'\${MODSEC_TAG}\''`) |
-| MODSEC_DEFAULT_PHASE2_ACTION | ModSecurity string with the contents for the default action in phase 2 (Default: `'phase:2,log,auditlog,pass,tag:\'\${MODSEC_TAG}\''`) |
-| CRS_ENABLE_TEST_MARKER | A boolean indicating whether to write test markers to the log file (Used for running the CRS test suite. Default: 0) |
+
+### CRS specific
+
+| Name     | Description|
+| -------- | ------------------------------------------------------------------- |
+| PARANOIA | An integer indicating the paranoia level (Default: `1`)               |
```

Comment:
Shall we sort this list?

Reply:
Maybe partially. I would still want things like `*PARANOIA*` to be grouped together, and `PARANOIA` should come before the other `*PARANOIA*` variables because they depend on it.

Reply:
I quickly tried sorting but it becomes clear quickly that multiple variables should be grouped together thematically, rather than alphabetically. Having those in the same table doesn't work well. How about one table per group?

Reply:
Probably it makes sense then to keep it simple. Leave it as it is now I guess?